### ISSUE TITLE
[organizing] joomla.org/media-contact.html need to be revisited

### ISSUE THREAD

**Issue by ufuk-avcu** [joomla/joomla-marketing#98]
We arrive so many support requests from this page https://www.joomla.org/media-contact.html to marketing@community.joomla.org . Maybe we can change the warning in the bottom to the top and make it more attracting?

**Comment by Joomv** [comment]
I think this page could do with a bit of a change anyway.  We should clearly signpost this page to the Forum, Github, Social etc.

**Comment by ufuk-avcu** [comment]
And Docs also...

**Comment by Sandra97** [comment]
You're assuming people read and follow the instructions.... You are dreamers!
Whatever we will do, it won't change anything, people will continue to use this form for their issues.

**Comment by mbabker** [comment]
Just a general note, you can put all the warnings you'd like on the page (or in this case, the information is made clear on multiple pages), but that won't slow it down.  I put a very clear notification on https://developer.joomla.org/security/contact-the-team.html before the form and the overwhelming majority of traffic to the security mailbox is still user support requests.

**Comment by Sandra97** [comment]
As I mentioned in Glip, this form was supposed to be for press requests only. But as soon as you add a contact form with high visibility, people will just use it for everything and won't care at all about notice. 
The other solution would be to remove the form and replace it by a text and an email address. Most people will be too lazy to write an email and you'll receive less messages for issues 😉

**Comment by cpaschen** [comment]
One other option, that would not reduce the amount of initial email, but might prevent some people from saying "I contacted them and they never replied" would be to add an auto-responder to that email address that thanks them for their email but tells them all that if they are seeking 'assistance' that they should ... (list the other resources) and indicates that if they are seeing media/marketing contact that it will be responded to within X days.
Also, I've seen some emails come in that people report problems with the j.org web site itself. That might be something to consider also 'handling' as most 'end users' have no idea how to report things via github (or how to use github at all).
BTW ... I'm still getting all these emails copied to me, so would be nice to remove that forwarding rule :-)

**Comment by cpaschen** [comment]
I take that back - I just looked at the responses that have come in today (and forwarded to me) and it looks like there is some weird forwarding of messages (forwarding info from one person who submitted something to someone else that submitted something - or some other strangeness).
This might need to have Luca or someone else figure out what is going on and get it cleaned up.

**Comment by cpaschen** [comment]
One other possibility would be to add a short video to that page (maybe if there is a video people will watch it)???

**Comment by parthlawate** [comment]
Can we make the following changes quickly and monitor over a period of 2 months ? 

1. Right now the Menu, Page title etc all say 'Contact' .. Can we at least change it to Press Inquiries or Media Enquiries ? 
2. Move the Notice to the top and rephrase it to say "This form is  strictly for Media queries. For general support about Joomla click here to go to the community forums. If you need help from Joomla professionals, click here to go to the Joomla Resource Directory"

**Comment by Sandra97** [comment]
I changed the title of the menu and edited the notice.
But it's totally useless. 
I can put the notice at the top, write it in red with a font-size of 100px, people will still use the form. Believe me, I have the same issue on other websites, people don't care about notices!

**Comment by Sandra97** [comment]
The only solution if nobody wants to take the time to answer these requests is to remove the contact form.

**Comment by parthlawate** [comment]
That was fast ! Thanks @Sandra97 I am closing this for now :) @ufuk-avcu lets monitor and see if this helps !
